docs/sphinx/source/user_guide/weather_data.rst

```diff
@@ -0,0 +1,159 @@
+.. _weatherdata:
+
+Weather data
+============
+
+Simulating the performance of a PV system requires irradiance and meteorological data
+as the inputs to a PV system model.  Weather datasets are available
+from many sources and in many formats.  The :py:mod:`pvlib.iotools` module
```

Comment:
Could we make a hyperlink to the iotools module for those who do not know what it is?